### DISCUSSION TITLE
SLower when cuda enabled?

### DISCUSSION THREAD

**Discussion by Coastchb** [coqui-ai/TTS#4066] [General Q&A]
I test with these three commands and get the results:
(1)python3 TTS/bin/synthesize.py   --model_name tts_models/en/vctk/vits --text "Thank you for your support, looking foward to continuing our cooperation next time" --speaker_idx p362 
(2)set --use_cuda False:   python3 TTS/bin/synthesize.py   --model_name tts_models/en/vctk/vits --text "Thank you for your support, looking foward to continuing our cooperation next time" --speaker_idx p362 --use_cuda False
(3)set --use_cuda True:   python3 TTS/bin/synthesize.py   --model_name tts_models/en/vctk/vits --text "Thank you for your support, looking foward to continuing our cooperation next time" --speaker_idx p362 --use_cuda True

and the result:
(1) 
 > Processing time: 0.8610982894897461
 > Real-time factor: 0.20372550733099679
 > Saving output to tts_output.wav
 > Done in 0.8714604377746582 seconds
(2)  
 > Processing time: 1.3045353889465332
 > Real-time factor: 0.30038643824426753
 > Saving output to tts_output.wav
 > Done in 1.3163387775421143 seconds
(3)
 > Processing time: 1.3771271705627441
 > Real-time factor: 0.3120956063035326
 > Saving output to tts_output.wav
 > Done in 1.3890950679779053 seconds

so, 
1、why (1) and (2) differs in inference time?
2、why it is much slower to inference with cuda?

**Comment by eginhard** [comment]
Judging from a single call is not really useful and may have a lot of variation. Best to load the model once, then synthesize e.g. 100 times to get an average time. Some details on the hardware would also be useful.
